connect/runtime/src/main/java/org/apache/kafka/connect/storage/KafkaOffsetBackingStore.java

```diff
@@ -248,7 +247,7 @@ public void start() {
                         + "support for source connectors, or upgrade to a newer Kafka broker version.";
             } else {
                 message = "When " + ConsumerConfig.ISOLATION_LEVEL_CONFIG + "is set to "
-                        + IsolationLevel.READ_COMMITTED.name().toLowerCase(Locale.ROOT)
+                        + IsolationLevel.READ_COMMITTED
```

Comment:
```suggestion
                        + IsolationLevel.READ_COMMITTED.toString()
```

Reply:
Oh my... I missed it.
Thank you for careful review!